lib/chef-acceptance/output_formatter.rb

```diff
@@ -0,0 +1,34 @@
+module ChefAcceptance
+  class OutputFormatter
```

Comment:
I think instead of this class we can have a `Stats.print` method which can print out all the stats we collect during the runs.